portfolio/src/main/webapp/script.js

```diff
@@ -80,3 +78,5 @@ function createListElement(text) {
   return liElement;
 
 }
+
```

Comment:
hmm, this shows two new lines at the end of the file, but there it also show there is no new line at the end of the file... Is there white space on these two new lines?